plugins/modules/cloud/misc/terraform.py

```diff
@@ -319,12 +319,24 @@ def remove_workspace(bin_path, project_path, workspace):
     _workspace_cmd(bin_path, project_path, 'delete', workspace)
 
 
-def build_plan(command, project_path, variables_args, state_file, targets, state, plan_path=None):
+def build_plan(command, project_path, variables_args, state_file, targets, state, APPLY_ARGS, plan_path=None):
```

Comment:
Here it is less a constant and more a parameter, so it shouldn't be upper-case.
```suggestion
def build_plan(command, project_path, variables_args, state_file, targets, state, apply_args, plan_path=None):
```

Reply:
Looks good, thanks.